charts/consul/values.yaml

```
@@ -25,7 +25,7 @@ global:
   name: null
 
   # The domain Consul will answer DNS queries for
-  # (see `-domain` (https://consul.io/docs/agent/options#_domain)) and the domain services synced from
+  # (see `-domain` (https://consul.io/docs/agent/options#domain)) and the domain services synced from
```

```suggestion
  # (see `-domain` (https://www.consul.io/docs/agent/config/cli-flags#_domain)) and the domain services synced from
```

The text here instructs the reader to refer to the [`-domain`](https://www.consul.io/docs/agent/config/cli-flags#_domain) flag, but then links to the [`domain`](https://www.consul.io/docs/agent/config/config-files#domain) configuration option.

It would be better to link to the parameter on the CLI flags docs when referencing a flag, and correspondingly to the config option docs when referencing a configuration option.